src/clp_logging/remote_handlers.py

```diff
@@ -0,0 +1,246 @@
+import base64
+import datetime
+import hashlib
+from pathlib import Path
+from typing import Any, Dict, List, Optional
+
+import boto3
+from botocore.exceptions import NoCredentialsError
+
+from clp_logging.handlers import CLPFileHandler
+
+
+class CLPRemoteHandler(CLPFileHandler):
+    """
+    Handles CLP file upload and comparison to AWS S3 bucket. Configuration of
+    AWS access key is required. Run command `aws configure`.
```

Comment:
> Configuration of AWS access key is required. Run command `aws configure`.

If I'm not wrong the credentials are stored in `~/.aws` by `aws configure`. Let's try to make the credentials directly configurable in our handler's configuration (probably via some __init__ parameter just like how the `HTTPHandler` requires a credentials parameter). Only when such credentials are not provided, we fallback to  credentials in `~/.aws`.